Twilio/Http/CurlClient.php

```diff
@@ -111,7 +111,7 @@ public function options($method, $url, $params = array(), $data = array(),
             CURLOPT_URL => $url,
             CURLOPT_HEADER => true,
             CURLOPT_RETURNTRANSFER => true,
-            CURLOPT_INFILESIZE => -1,
+            CURLOPT_INFILESIZE => Null,
```

Comment:
What happens if we don't specify this parameter? What will it default to?

Secondly, shouldn't it be `null`?

Reply:
it looks like `curl_setopt_array` expects `CURLOPT_INFILESIZE` value to be  a `long`, so specifying `Null` may have the same result as 0 or removing the line.  (I tried to find the answer in PHP src, but got lost in Zend types.)